test/hlx_server_test.js

```diff
@@ -776,11 +778,32 @@ describe('Helix Server', () => {
     }
   });
 
+  it('user provided action parameters get passed to Server', async () => {
```

Comment:
the ultimate test would be to read the `action.params.SOMETHING` in a script and respond it with the http response.